Makefile

```diff
@@ -1,9 +1,9 @@
 default: all
 
 PDF_FILES = index.pdf
-HTML_FILES = index.html 404.html
```

Comment:
@ellert 
I either misunderstood how the build works or just forgot to remove this after creating the HTML file, don't remember. Good that you noticed this.